src/Security/PermissionVoterCommandBusSecurity.php

```diff
@@ -13,7 +13,7 @@ final class PermissionVoterCommandBusSecurity implements CommandBusSecurity
     private PermissionVoter $permissionVoter;
 
     public function __construct(
-        ?string $userId,
+        string $userId,
```

Comment:
Is this change needed in the context of this pull request?

Reply:
No, but i am really sure this is a bug, for 2 reasons:
- It makes no sense that the property declaration and the constructor are different.
- When I did the opposite - make it both nullable, it just failed with other dependencies down the line.

If you want I can cherrypick this into the worlds smallest PR? ;-)

Reply:
But the `getId` method from `CurrentUser` returns `?string` see https://github.com/cultuurnet/udb3-backend/blob/b491c80878447306225bac139da04f6e10deccc3/src/User/CurrentUser.php#L24 so method also has a wrong return type?